rockcraft.yaml

```diff
@@ -0,0 +1,163 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+---
+
+name: charmed-ranger-rock
+base: ubuntu@22.04
+version: 2.4.0-22.04-edge
+summary: Charmed ranger ROCK OCI
+description: |
+  Apache Ranger™ is a framework to enable, monitor and
+  manage comprehensive data security across the Hadoop platform.
+license: Apache-2.0
+
+platforms:
+  amd64:
+
+parts:
+  base:
+    plugin: nil
+    source: "https://downloads.apache.org/ranger/2.4.0/apache-ranger-2.4.0.tar.gz" # yamllint disable-line
+    build-packages:
+      - build-essential
+      - curl
+      - gcc
```

Comment:
do you need gcc, gnupg, patch, git?

Reply:
I need git for building the (I think it's this one >) `Apache Ranger Distribution` tar. The others I will remove.